### DISCUSSION TITLE
immich.app availability

### DISCUSSION THREAD

**Discussion by HELPY4** [immich-app/immich#14285] [General]
Quite often I am unable to open the Immich documentation. I don't get any specific error, just ‘Timeout’ (more on the screenshots).
I can send any relevant information that may help to solve the problem.
Thanks in advance :)
![image](https://github.com/user-attachments/assets/39368b79-5fb6-4aab-acc1-7677a25084fe)
![image](https://github.com/user-attachments/assets/2b6cc319-44e2-4624-b5aa-6306ae0ecf32)
![image](https://github.com/user-attachments/assets/c1fd551c-70f9-4248-9e67-2cee94aac7c5)
![image](https://github.com/user-attachments/assets/c4564899-d6ed-4d53-aed1-f26768fabe7f)

**Comment by alextran1502** [comment]
Are you using VPN and do you mind letting us know where do you accessing the site from?

**Reply by HELPY4** [reply]
No, I don't use a VPN. Sorry I didn't write it straight away, I live in Russia

**Reply by bo0tzz** [reply]
Are you able to reach other websites hosted through cloudflare?

**Reply by HELPY4** [reply]
Yes
